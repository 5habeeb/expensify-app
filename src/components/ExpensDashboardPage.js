import React from 'react';
import ExpensesList from './ExpensesList';
import ExpensesListFilters from './ExpensesListFilters';

const ExpensDashboardPage = () => (
    <div>
        <ExpensesListFilters />
        <ExpensesList />
    </div>
);

export default ExpensDashboardPage;