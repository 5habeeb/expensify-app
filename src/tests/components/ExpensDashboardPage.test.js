import React from 'react';
import { shallow } from 'enzyme'
import ExpensDashboardPage  from '../../components/ExpensDashboardPage';

test('should render ExpensDashboardPage', () => {
    const wrapper = shallow(<ExpensDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});