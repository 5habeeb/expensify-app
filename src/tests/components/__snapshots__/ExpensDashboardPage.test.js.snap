// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensDashboardPage 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpensesListFilters) />
  <Connect(ExpensesList) />
</div>
`;
